### PR TITLE
[KIECLOUD-171] - Update Application templates and imagestreams on OCP…

### PR DESCRIPTION
**Pull request by spolti**
… for RHDS and RHIPS

Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

File: official/decisionserver/imagestreams/jboss-decisionserver64-openshift-rhel7.json [hunk 1]
```diff
@@ -97,6 +97,48 @@
...
                     "type": "Local"
                 }
+            },
+            {
+                "annotations": {
+                    "description": "Red Hat JBoss BRMS 6.4 decision server S2I images.",
+                    "iconClass": "icon-decisionserver",
+                    "openshift.io/display-name": "Red Hat JBoss BRMS 6.4 decision server",
+                    "sampleContextDir": "decisionserver/hellorules",
+                    "sampleRef": "1.3",
```
Comment (adambkaplan) [inline official/decisionserver/imagestreams/jboss-decisionserver64-openshift-rhel7.json]: wrong version number?

File: official/decisionserver/imagestreams/jboss-decisionserver64-openshift-rhel7.json [hunk 2]
```diff
@@ -97,6 +97,48 @@
...
+                    "type": "Local"
+                }
+            },
+            {
+                "annotations": {
+                    "description": "Red Hat JBoss BRMS 6.4 decision server S2I images.",
+                    "iconClass": "icon-decisionserver",
+                    "openshift.io/display-name": "Red Hat JBoss BRMS 6.4 decision server",
+                    "sampleContextDir": "decisionserver/hellorules",
+                    "sampleRef": "1.4",
```
Comment (adambkaplan) [inline official/decisionserver/imagestreams/jboss-decisionserver64-openshift-rhel7.json]: wrong version number?

File: official/processserver/imagestreams/jboss-processserver64-openshift-rhel7.json [hunk 3]
```diff
@@ -97,6 +97,48 @@
...
                     "type": "Local"
                 }
+            },
+            {
+                "annotations": {
+                    "description": "Red Hat JBoss BPM Suite 6.4 intelligent process server S2I images.",
+                    "iconClass": "icon-processserver",
+                    "openshift.io/display-name": "Red Hat JBoss BPM Suite 6.4 intelligent process server",
+                    "sampleContextDir": "processserver/library",
+                    "sampleRef": "1.3",
```
Comment (adambkaplan) [inline official/processserver/imagestreams/jboss-processserver64-openshift-rhel7.json]: wrong version?

File: official/processserver/imagestreams/jboss-processserver64-openshift-rhel7.json [hunk 4]
```diff
@@ -97,6 +97,48 @@
...
+                    "type": "Local"
+                }
+            },
+            {
+                "annotations": {
+                    "description": "Red Hat JBoss BPM Suite 6.4 intelligent process server S2I images.",
+                    "iconClass": "icon-processserver",
+                    "openshift.io/display-name": "Red Hat JBoss BPM Suite 6.4 intelligent process server",
+                    "sampleContextDir": "processserver/library",
+                    "sampleRef": "1.4",
```
Comment (adambkaplan) [inline official/processserver/imagestreams/jboss-processserver64-openshift-rhel7.json]: wrong version?

Comment (spolti) [reply official/processserver/imagestreams/jboss-processserver64-openshift-rhel7.json]: Hi, not exactly, it is just the quickstart version used in the ImageStream.

**Comment by spolti** [comment]
Hi, can this be merged?

**Comment by spolti** [comment]
Hi folks, any update?

**Comment by bparees** [comment]
@adambkaplan bump

**Pull request merged**